pkg/gardenlet/apis/config/types.go

```diff
@@ -309,10 +309,23 @@ type FluentBit struct {
 	OutputSection *string
 }
 
+// Loki contains configuration for the Loki.
+type Loki struct {
+	// GardenLoki contains configuration for the Loki in garden namespace.
+	GardenLoki *GardenLoki
```

Comment:
While you are touching these lines, what do you think to add the same configuration options but for the shoot lokis?

Reply:
Not exactly the same but, am planning to make the Loki ConfigMap adjustable via the gardelet configuration.
And because the central Loki is in much higher pressure the configuration could be different.